clients/admin-ui/src/features/common/Icon/ArrowDownLine.tsx

```diff
@@ -0,0 +1,7 @@
+import { createIcon } from '@chakra-ui/react';
```

Comment:
Definitely should be importing this from `fidesui/react`, fixed. As for alt text, SVGs as a rule don't support them, so Chakra doesn't support them.

It would be good to add a `<desc />` or `<title />` tag or something, but we would have to wrap the `createIcon` function or submit a PR to Chakra to do that. I don't think it's a priority for right now. Definitely something to note for the future, though.